static/app/views/automations/detail.tsx

```diff
@@ -246,7 +250,12 @@ function Actions({automation}: {automation: Automation}) {
       <Button priority="default" size="sm" onClick={toggleDisabled} busy={isUpdating}>
         {automation.enabled ? t('Disable') : t('Enable')}
       </Button>
-      <LinkButton to="edit" priority="primary" icon={<IconEdit />} size="sm">
+      <LinkButton
+        to={`${makeAutomationDetailsPathname(organization.slug, automation.id)}edit/`}
```

Comment:
We could also export a new `makeAutomationEditPathname` if we wanted to avoid appending `edit`